platform/viewer/src/utils/reports/exportPDF.js

```diff
@@ -0,0 +1,105 @@
+import { utils } from '@ohif/core';
+import csTools from 'cornerstone-tools';
+import cornerstone from 'cornerstone-core';
+
+import MeasurementReport from './MeasurementReport';
```

Comment:
Since this will pull in jspdf, maybe we can dynamically import the export functionality (somewhere, not sure it's best here), since jspdf is pretty big